ui/app/AppLayouts/Wallet/components/SendModalContent.qml

```diff
@@ -47,7 +45,17 @@ Item {
             passwordValidationError = ""
         }
 
-        return passwordValidationError === "" && toValidationError === "" && amountValidationError === "" && isRecipientValid && isAssetAndAmountValid
+        return passwordValidationError === "" && isRecipientValid && isAssetAndAmountValid
+    }
+
+    function showPreview() {
+        pvwTransaction.visible = true
+        txtAmount.visible = selectFromAccount.visible = selectRecipient.visible = txtPassword.visible = gasSelector.visible = false
+    }
+
+    function showInputs() {
+        pvwTransaction.visible = false
+        txtAmount.visible = selectFromAccount.visible = selectRecipient.visible = txtPassword.visible = gasSelector.visible = true
```

Comment:
Would it be easier to create a surrounding Item as a group and them just hide and show that?

Reply:
Good idea, but this is all going to change once the modal is composed as per the design so I didn't spend any time on it and likely doesn't need to change here.